.github/actions/install-rake-dependencies/action.yml

```diff
@@ -0,0 +1,12 @@
+name: Rake dependencies installer
+jobs:
+description: 'Install Rake dependencies like npm'
```

Comment:
I don't think this is an accurate description of this step. Can it just say "Runs rake install and rake build"?